.github/workflows/build.yaml

```diff
@@ -74,7 +74,20 @@ jobs:
 
     # Audit all currently installed packages for security vulnerabilities.
     - name: Audit installed packages
-      run: make audit
+      id: audit-packages
+      continue-on-error: true
+      run: make --silent audit > vulnerabilities.txt
+
+    # Upload the vulnerabilities file output.
+    - name: Upload Artifact
+      id: upload-audit-artifact
+      uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb #v3.1.1
+      with:
+        name: vulnerabilities.txt
+        path: .
+        if-no-files-found: error
+        retention-days: 1
+      if: steps.audit-packages.outputs.exit_code == 1
```

Comment:
I wonder if `steps.audit-packages.conclusion == 'failure'` is more reliable because any non-zero exit code is considered a failure (and we check only for `1`<sup>*</sup>)?

—————
<sup>*</sup> At first glance, `1` seems to be the only exit code that [`pip-audit` returns](https://github.com/pypa/pip-audit/blob/main/pip_audit/_cli.py).